src/features/components/templates/tabs/menu/index.tsx

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { Localize } from 'components/localization'
+import FlexBox from 'features/components/atoms/flex-box'
+import Tab from 'features/components/atoms/tab'
+import { TString } from 'types/generics'
+
+export type TabMenuProps = {
+    class_name?: string
```

Comment:
It would look good.

```suggestion
    className?: string
```

Reply:
props properties should name in snake_case as far as I know